l10n_be_national_number/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import res_partner
+from . import res_partner_id_category
```

Comment:
I do not see this file in the PR. Did you forget to include it ?

Reply:
fixed